### PR TITLE
GitHub actions CI

### PR DESCRIPTION
**Pull request by dzenanz**
Closes #106.

### CODE REVIEW

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [ubuntu-18.04, windows-2019, macos-10.15]
+        include:
+          - os: ubuntu-18.04
+            c-compiler: "gcc"
+            cxx-compiler: "g++"
+            itk-git-tag: "v5.1.1"
```
Comment (thewtex) [inline .github/workflows/build-test-package.yml:16]: We can update these to a hash on master to add your remote module improvements.

File: .github/workflows/build-test-package.yml [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+            cmake-build-type: "MinSizeRel"
+          - os: windows-2019
+            c-compiler: "cl.exe"
+            cxx-compiler: "cl.exe"
+            itk-git-tag: "v5.1.1"
```
Comment (thewtex) [inline .github/workflows/build-test-package.yml:21]: We can update these to a hash on master to add your remote module improvements.

File: .github/workflows/build-test-package.yml [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+            cmake-build-type: "Release"
+          - os: macos-10.15
+            c-compiler: "clang"
+            cxx-compiler: "clang++"
+            itk-git-tag: "v5.1.1"
```
Comment (thewtex) [inline .github/workflows/build-test-package.yml:26]: We can update these to a hash on master to add your remote module improvements.

**Review by thewtex — CHANGES_REQUESTED** [review]
A few inline comments that should help fix the C++ builds.

Could you please push the topic branch to `KitwareMedical/ITKUltrasound` and create a pull request from that branch instead of the forked repository? That will enable testing builds.

Extra steps will be required for the Python package builds, be we can take those on in a per-platform basis after the C++ tests are passing.

**Comment by dzenanz** [comment]
Opened #121.

**Pull request merged**